app/controllers/health_cards_controller.rb

```diff
@@ -19,12 +19,16 @@ def scan; end
 
   def qr_contents
     contents = JSON.parse(params[:qr_contents])
+    jws_payload = HealthCards::Chunking.get_payload_from_qr contents
+    bundle = FHIR.from_contents(jws_payload['vc']['credentialSubject']['fhirBundle'].to_json)
 
-    Rails.logger.info(contents.join("\n"))
+    patient_resource = bundle.entry.select { |e| e.resource.is_a?(FHIR::Patient) }[0].resource
```

Comment:
I get this error when using [example 2](https://smarthealth.cards/examples/#example-2):

![Screen Shot 2021-04-09 at 9 59 32 AM](https://user-images.githubusercontent.com/15969967/114192441-79b4f080-991b-11eb-89b4-335ff99631f5.png)

Reply:
Yea I'm expecting a `Patient` resource in the fhir bundle. [Example 2](https://smarthealth.cards/examples/example-02-a-fhirBundle.json) doesn't seem to have one. Do you think I should just display the whole vc if no patient is available?

Reply:
Sure.

Reply:
I think this makes the most sense. Fill in the patient and immunization into the nice UI if they are available, but default to displaying the VC if they are not available since they aren't actually required by SMART Health Cards.

Reply:
I think we are specifically doing a Covid-19 implementation, so if `Patient` is required, it's ok to display a message explaining that it isn't a valid VC, especially as a first pass. It's just not ok to return a 500.

Reply:
Changed to display jws payload if `Patient` isn't found in bundle.